app/presenters/recommended_feed_item_presenter.rb

```diff
@@ -0,0 +1,9 @@
+class RecommendedFeedItemPresenter < FeedItemPresenter
+  def name
+    'RecommendedFeedItem'
+  end
+
+  def to_partial_path
+    "shared/#{name.underscore}"
+  end
+end
```

Comment:
DRY. Makes use of inheritance, separation of concern, clean views. Our recommended feed item presenter "wraps" a recommended (starred) feed item and provides additional functionality to be used in our subsequent views. Other members (#name and #to_partial_path) allows the rails to identify what partials need to be used to render this object.